mhs-reference-implementation/mhs/routing/sds.py

```diff
@@ -0,0 +1,117 @@
+import asyncio
+import logging
+import ssl
+import pathlib
+from definitions import ROOT_DIR
+import ldap3
+import mhs.routing.routing_exception as re
+
+NHS_SERVICES_BASE = "ou=services, o=nhs"
+
+MHS_OBJECT_CLASS = "nhsMhs"
+AS_OBJECT_CLASS = "nhsAs"
+MHS_PARTY_KEY = 'nhsMHSPartyKey'
+
+# These need replacing as the appropriate TLS certs are received for PTL
+PRIVATE_KEY = pathlib.Path(ROOT_DIR) / 'data' / 'certs' / 'client.key'
+LOCAL_CERT_FILE = pathlib.Path(ROOT_DIR) / 'data' / 'certs' / 'client.cert'
+CA_CERTS_FILE = pathlib.Path(ROOT_DIR) / 'data' / 'certs' / 'client.cert'
```

Comment:
If this is the same as the previous constant and may change, then simpler to do:
```suggestion
CA_CERTS_FILE = LOCAL_CERT_FILE
```
for the timebeing

Reply:
Ultimately these will be different files, I'll update the paths to demonstrate this and I've added a constant holding the certs dir path

Reply:
Removed these completely actually